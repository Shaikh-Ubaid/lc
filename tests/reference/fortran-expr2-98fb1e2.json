{
    "basename": "fortran-expr2-98fb1e2",
    "cmd": "lc --no-color --show-fortran {infile}",
    "infile": "tests/expr2.c",
    "infile_hash": "11cb114715d67a4763454167abc900089b2119ecf18a980d18a30322",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "fortran-expr2-98fb1e2.stdout",
    "stdout_hash": "79877dd33c7f4304ceefe7e697e1d6840b3e72e9b5e84cf470b747c9",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}