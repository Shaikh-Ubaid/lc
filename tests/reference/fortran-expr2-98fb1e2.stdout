program main_program
implicit none
integer(4) :: exit_code
exit_code = main()

contains

integer(4) function main() result(__return_var)
    integer(4) :: x
    x = (2 + 3)*5
    print *, x
    __return_var = 0
    return
end function main

end program main_program
